packages/edit-site/src/components/canvas-loader/index.js

```diff
@@ -34,7 +34,9 @@ export default function CanvasLoader( { id } ) {
 	return (
 		<div className="edit-site-canvas-loader">
 			<Theme accent={ indicatorColor } background={ backgroundColor }>
-				<ProgressBar id={ id } max={ total } value={ elapsed } />
+				<div style={ { width: '160px' } }>
```

Comment:
I don't like having to forcedly add these wrapper divs in the current consumers, I can't avoid thinking a self-contained default would be better here, but I'm looking forward to reading your thoughts. However, it does fix the issue [here](https://github.com/WordPress/gutenberg/pull/61062#pullrequestreview-2073018536) by making it look exactly like the current version on `trunk`.

Reply:
Doesn't necessarily have to be a wrapper div. Another standard way would be to pass your own class name:

```
.my-progress-bar {
  width: 160px;
}

<ProgressBar className="my-progress-bar" />
```

In either case I would prefer a CSS class rather than an inline style, since there is no reason for this to be inline.

Reply:
Cool, will move it over to a CSS class there 👍🏻

Reply:
Done in https://github.com/WordPress/gutenberg/pull/61062/commits/bc95336241c9240c4820d61703ac9052f652d896.